### DISCUSSION TITLE
client?

### DISCUSSION THREAD

**Discussion by Rapiddox** [TrinityCore/TrinityCore#28763] [Q&A] [answered]
Hi this is only the server-side of the wow? Where can I find the client

**Comment by Jinnaix** [comment]
For Master Branch from Blizzard, for 335 from searching via google.
We do not share clients.

**Comment by Ovahlord** [comment, marked as answer]
that's something you gotta find out on your own because we may not distribute any copyrighted material for obvious reasons.
